react-exercises/modal-react/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./App.css";
+import LoginModal from "./component/LoginModal";
+import SignUpModal from "./component/SignUpModal";
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
you can do without constructor and binding function. 
For that add arrow function in all your handle method .
you can call props directly by using only "this.props"

Reply:
Thank you, I have updated the functions.